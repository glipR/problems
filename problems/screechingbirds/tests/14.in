32 443
55 60 cmbigdilh
89 86 qqtyqetsgi
50 15 jpqmvq
61 57 prvfbdpok
52 24 oqpndorxoddumfsxnspbjy
80 43 mequelcjjyjwqlublkurgill
41 74 ttfgyyarsqdrkuz
6 20 azutvlcnekagswkaqyc
63 10 fgtnpbk
72 38 fpmpr
91 89 imzntlfarfwllqzldqwjs
39 12 fbgtnswof
30 10 rxtzbysmms
30 91 nrkyxjb
100 68 upsraewxkl
91 72 ebujqneuwoajzqfrtz
98 3 qpfdjiovzcvwfjobqaneowd
92 51 yewoirketsvwkcxiwdaplhip
44 3 vgyxmppvnpmfnualutbtnfly
84 39 fjwymlcionbvtmktboaatychn
74 41 kwbhfrdbplosvvb
43 59 qnzevrnuypqyzpiopotbnc
46 30 dqcbeeatcinvliapnjfumcu
40 69 xmqpuxhxewt
21 47 xmmsplfag
88 59 dpfhgbtvexirysaqmrovxzoru
38 88 rrmrbplpzwpcwxxrh
23 58 jfxniodeaimtcg
86 36 gtsdyuklqqgyumtacwhamk
13 61 eqyjcdrfnvgfgshqaivxgdwfj
71 93 jaeqygoenvju
63 16 doukujgefymmbqd