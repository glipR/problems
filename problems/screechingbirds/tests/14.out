azutvlcnekagswkaqyc
eqyjcdrfnvgfgshqaivxgdwfj
xmmsplfag
jfxniodeaimtcg
azutvlcnekagswkaqyc
rxtzbysmms
nrkyxjb
rrmrbplpzwpcwxxrh
fbgtnswof
rxtzbysmms
xmqpuxhxewt
ttfgyyarsqdrkuz
qnzevrnuypqyzpiopotbnc
vgyxmppvnpmfnualutbtnfly
azutvlcnekagswkaqyc
dqcbeeatcinvliapnjfumcu
vgyxmppvnpmfnualutbtnfly
jpqmvq
rxtzbysmms
vgyxmppvnpmfnualutbtnfly
fbgtnswof
oqpndorxoddumfsxnspbjy
vgyxmppvnpmfnualutbtnfly
cmbigdilh
vgyxmppvnpmfnualutbtnfly
vgyxmppvnpmfnualutbtnfly
rxtzbysmms
prvfbdpok
vgyxmppvnpmfnualutbtnfly
fgtnpbk
fbgtnswof
doukujgefymmbqd
jpqmvq
vgyxmppvnpmfnualutbtnfly
azutvlcnekagswkaqyc
vgyxmppvnpmfnualutbtnfly
xmmsplfag
rxtzbysmms
vgyxmppvnpmfnualutbtnfly
jaeqygoenvju
fpmpr
fgtnpbk
vgyxmppvnpmfnualutbtnfly
kwbhfrdbplosvvb
eqyjcdrfnvgfgshqaivxgdwfj
fbgtnswof
oqpndorxoddumfsxnspbjy
dqcbeeatcinvliapnjfumcu
vgyxmppvnpmfnualutbtnfly
doukujgefymmbqd
jpqmvq
mequelcjjyjwqlublkurgill
rxtzbysmms
vgyxmppvnpmfnualutbtnfly
jfxniodeaimtcg
fgtnpbk
vgyxmppvnpmfnualutbtnfly
fjwymlcionbvtmktboaatychn
azutvlcnekagswkaqyc
vgyxmppvnpmfnualutbtnfly
gtsdyuklqqgyumtacwhamk
fbgtnswof
dpfhgbtvexirysaqmrovxzoru
qqtyqetsgi
vgyxmppvnpmfnualutbtnfly
rxtzbysmms
imzntlfarfwllqzldqwjs
ebujqneuwoajzqfrtz
yewoirketsvwkcxiwdaplhip
vgyxmppvnpmfnualutbtnfly
fgtnpbk
jpqmvq
vgyxmppvnpmfnualutbtnfly
doukujgefymmbqd
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
fbgtnswof
oqpndorxoddumfsxnspbjy
rxtzbysmms
upsraewxkl
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
qnzevrnuypqyzpiopotbnc
fgtnpbk
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
azutvlcnekagswkaqyc
dqcbeeatcinvliapnjfumcu
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
xmqpuxhxewt
jpqmvq
fpmpr
rxtzbysmms
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
fbgtnswof
doukujgefymmbqd
fgtnpbk
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
cmbigdilh
ttfgyyarsqdrkuz
kwbhfrdbplosvvb
xmmsplfag
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
prvfbdpok
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
rxtzbysmms
nrkyxjb
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
gtsdyuklqqgyumtacwhamk
mequelcjjyjwqlublkurgill
fgtnpbk
fbgtnswof
fjwymlcionbvtmktboaatychn
oqpndorxoddumfsxnspbjy
jpqmvq
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
azutvlcnekagswkaqyc
rrmrbplpzwpcwxxrh
doukujgefymmbqd
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
rxtzbysmms
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
fgtnpbk
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
fbgtnswof
eqyjcdrfnvgfgshqaivxgdwfj
dqcbeeatcinvliapnjfumcu
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
jfxniodeaimtcg
jpqmvq
rxtzbysmms
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
fgtnpbk
qpfdjiovzcvwfjobqaneowd
yewoirketsvwkcxiwdaplhip
vgyxmppvnpmfnualutbtnfly
doukujgefymmbqd
azutvlcnekagswkaqyc
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
fbgtnswof
dpfhgbtvexirysaqmrovxzoru
oqpndorxoddumfsxnspbjy
fpmpr
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
rxtzbysmms
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
fgtnpbk
jpqmvq
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
kwbhfrdbplosvvb
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
gtsdyuklqqgyumtacwhamk
fbgtnswof
doukujgefymmbqd
rxtzbysmms
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
qnzevrnuypqyzpiopotbnc
fjwymlcionbvtmktboaatychn
xmmsplfag
fgtnpbk
ebujqneuwoajzqfrtz
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
jaeqygoenvju
mequelcjjyjwqlublkurgill
azutvlcnekagswkaqyc
dqcbeeatcinvliapnjfumcu
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
upsraewxkl
jpqmvq
rxtzbysmms
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
fbgtnswof
oqpndorxoddumfsxnspbjy
fgtnpbk
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
cmbigdilh
qqtyqetsgi
prvfbdpok
doukujgefymmbqd
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
xmqpuxhxewt
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
imzntlfarfwllqzldqwjs
rxtzbysmms
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
fgtnpbk
fbgtnswof
jpqmvq
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
azutvlcnekagswkaqyc
fpmpr
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
ttfgyyarsqdrkuz
rxtzbysmms
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
doukujgefymmbqd
fgtnpbk
qpfdjiovzcvwfjobqaneowd
yewoirketsvwkcxiwdaplhip
vgyxmppvnpmfnualutbtnfly
gtsdyuklqqgyumtacwhamk
fbgtnswof
oqpndorxoddumfsxnspbjy
dqcbeeatcinvliapnjfumcu
eqyjcdrfnvgfgshqaivxgdwfj
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
kwbhfrdbplosvvb
jfxniodeaimtcg
jpqmvq
rxtzbysmms
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
fjwymlcionbvtmktboaatychn
fgtnpbk
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
azutvlcnekagswkaqyc
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
dpfhgbtvexirysaqmrovxzoru
fbgtnswof
doukujgefymmbqd
mequelcjjyjwqlublkurgill
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
xmmsplfag
rxtzbysmms
nrkyxjb
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
fgtnpbk
rrmrbplpzwpcwxxrh
jpqmvq
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
fbgtnswof
oqpndorxoddumfsxnspbjy
rxtzbysmms
qnzevrnuypqyzpiopotbnc
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
fgtnpbk
doukujgefymmbqd
fpmpr
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
azutvlcnekagswkaqyc
dqcbeeatcinvliapnjfumcu
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
jpqmvq
rxtzbysmms
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
gtsdyuklqqgyumtacwhamk
fbgtnswof
prvfbdpok
fgtnpbk
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
cmbigdilh
ebujqneuwoajzqfrtz
upsraewxkl
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
kwbhfrdbplosvvb
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
doukujgefymmbqd
rxtzbysmms
fjwymlcionbvtmktboaatychn
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
fgtnpbk
fbgtnswof
oqpndorxoddumfsxnspbjy
jpqmvq
qpfdjiovzcvwfjobqaneowd
yewoirketsvwkcxiwdaplhip
vgyxmppvnpmfnualutbtnfly
azutvlcnekagswkaqyc
xmqpuxhxewt
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
rxtzbysmms
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
mequelcjjyjwqlublkurgill
fgtnpbk
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
fbgtnswof
jfxniodeaimtcg
doukujgefymmbqd
dqcbeeatcinvliapnjfumcu
xmmsplfag
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
eqyjcdrfnvgfgshqaivxgdwfj
jaeqygoenvju
jpqmvq
rxtzbysmms
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
qqtyqetsgi
fpmpr
ttfgyyarsqdrkuz
fgtnpbk
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
dpfhgbtvexirysaqmrovxzoru
azutvlcnekagswkaqyc
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
gtsdyuklqqgyumtacwhamk
fbgtnswof
oqpndorxoddumfsxnspbjy
imzntlfarfwllqzldqwjs
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
rxtzbysmms
doukujgefymmbqd
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
fgtnpbk
jpqmvq
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
fbgtnswof
fjwymlcionbvtmktboaatychn
kwbhfrdbplosvvb
qnzevrnuypqyzpiopotbnc
rxtzbysmms
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
fgtnpbk
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
azutvlcnekagswkaqyc
dqcbeeatcinvliapnjfumcu
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
doukujgefymmbqd
prvfbdpok
jpqmvq
rxtzbysmms
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
fbgtnswof
oqpndorxoddumfsxnspbjy
fgtnpbk
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
cmbigdilh
mequelcjjyjwqlublkurgill
qpfdjiovzcvwfjobqaneowd
yewoirketsvwkcxiwdaplhip
vgyxmppvnpmfnualutbtnfly
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
fpmpr
rxtzbysmms
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
rrmrbplpzwpcwxxrh
gtsdyuklqqgyumtacwhamk
fgtnpbk
fbgtnswof
nrkyxjb
xmmsplfag
doukujgefymmbqd
upsraewxkl
jpqmvq
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
azutvlcnekagswkaqyc
ebujqneuwoajzqfrtz
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
rxtzbysmms
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
fgtnpbk
jfxniodeaimtcg
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
fbgtnswof
oqpndorxoddumfsxnspbjy
dqcbeeatcinvliapnjfumcu
xmqpuxhxewt
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
fjwymlcionbvtmktboaatychn
eqyjcdrfnvgfgshqaivxgdwfj
doukujgefymmbqd
jpqmvq
rxtzbysmms
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
kwbhfrdbplosvvb
fgtnpbk
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
dpfhgbtvexirysaqmrovxzoru
azutvlcnekagswkaqyc
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
fbgtnswof
qpfdjiovzcvwfjobqaneowd
vgyxmppvnpmfnualutbtnfly
